include/bluetooth/mesh/light_ctrl_srv.h

```diff
@@ -20,29 +20,46 @@
 #include <bluetooth/mesh/gen_onoff_srv.h>
 #include <bluetooth/mesh/lightness_srv.h>
 #include <bluetooth/mesh/model_types.h>
+#include <bluetooth/mesh/light_ctrl_regulator.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 struct bt_mesh_light_ctrl_srv;
 
-/** @def BT_MESH_LIGHT_CTRL_SRV_INIT
+/** @def BT_MESH_LIGHT_CTRL_SRV_INIT_WITH_REG
  *
- *  @brief Initialization parameters for @ref bt_mesh_light_ctrl_srv.
+ *  @brief Initialization parameters for @ref bt_mesh_light_ctrl_srv with custom regulator.
  *
  *  @param[in] _lightness_srv Pointer to the @ref bt_mesh_lightness_srv this
  *                            server controls.
+ *  @param[in] _reg_cb        Pointer to the @ref bt_mesh_light_ctrl_srv_reg_cb used for
+ *                            interfacing with the regulator
+ *  @param[in] _reg_data      Void pointer to user data which will be passed to regulator
+ *                            callbacks.
  */
-#define BT_MESH_LIGHT_CTRL_SRV_INIT(_lightness_srv)                            \
+#define BT_MESH_LIGHT_CTRL_SRV_INIT_WITH_REG(_lightness_srv,                   \
+					     _reg_cb,                          \
+					     _reg_data)                        \
 	{                                                                      \
 		.cfg = BT_MESH_LIGHT_CTRL_SRV_CFG_INIT,                        \
 		.onoff = BT_MESH_ONOFF_SRV_INIT(                               \
 			&_bt_mesh_light_ctrl_srv_onoff),                       \
 		.lightness = _lightness_srv,                                   \
-		BT_MESH_LIGHT_CTRL_SRV_REG_INIT                                \
+		BT_MESH_LIGHT_CTRL_SRV_REG_INIT(_reg_cb, _reg_data)            \
 	}
 
+/** @def BT_MESH_LIGHT_CTRL_SRV_INIT
+ *
+ *  @brief Initialization parameters for @ref bt_mesh_light_ctrl_srv without custom regulator.
```

Comment:
```suggestion
 *  @brief Initialization parameters for @ref bt_mesh_light_ctrl_srv with default regulator.
```
Is this a little more descriptive?